pandas/tests/series/test_combine_concat.py

```diff
@@ -118,61 +61,45 @@ def get_result_type(dtype, dtype2):
                 result = pd.concat([Series(dtype=dtype), Series(dtype=dtype2)]).dtype
                 assert result.kind == expected
 
-    def test_concat_empty_series_dtypes(self):
+    @pytest.mark.parametrize(
+        "left,right,expected",
+        [
+            # booleans
+            (np.bool_, np.int32, np.int32),
+            (np.bool_, np.float32, np.object_),
+            # datetime-like
+            ("m8[ns]", np.bool, np.object_),
+            ("m8[ns]", np.int64, np.object_),
+            ("M8[ns]", np.bool, np.object_),
+            ("M8[ns]", np.int64, np.object_),
+            # categorical
+            ("category", "category", "category"),
+            ("category", "object", "object"),
+        ],
+    )
```

Comment:
again keep test refactoring separate from moving things. (in future)

Reply:
good reminder